components/header-bar/src/online-status.js

```diff
@@ -18,15 +22,24 @@ const useOnlineStatusInfo = ({ online, lastOnline }) => {
             relativeTime: moment(lastOnline).fromNow(),
         })
     }
-    // todo: in the future, support 'CUSTOM' option
+
+    if (onlineStatusMessage) {
+        return onlineStatusMessage
```

Comment:
I've a question about this functional I think I have already partial covered this in a conversation with @KaiVandivier but would be good to get clarity.

Rummaging though the tickets I found a design that looked like this..

![message with lastonline](https://user-images.githubusercontent.com/110382464/196666793-9bbdb23c-ba82-4fe8-b3db-bd1387c9919f.png)

Mentioned in https://docs.google.com/document/d/1BRU5yoiZDcgVDLJeFXih8uoI2DnjONOApbgwi1qU1FQ/edit#heading=h.7sxyqvjcxlix

So I wonder if the `onlineStatusMessage` needs to display alongside time `lastOnline`

2 main questions from this

1. Do we need to retain the the config check of `headerbar?.onlineStatusInfo === 'LAST_ONLINE'` for backward compatibility?
2. If we are retaining that config check, given the design I mention above should `onlineStatusMessage` act more like a prefix? So rather than either `lastOnline` or `onlineStatusMessage` or would be (pseudo code)`<>{onlineStatusMessage} {lastOnline}</>` so both could be potentially displayed? The other option here is that if you set `onlineStatusMessage` with a custom message and want the last online time added that you would have to specify that yourself in the custom message.

Reply:
Awesome, thanks @KaiVandivier thats make sense and simplifies whats happening here too :-)